Example/SnapshotTests/Backpack_SnapshotTests-Bridging-Header.h

```diff
@@ -15,15 +15,3 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
```

Comment:
Is there any point in this file anymore?

Reply:
I think I added in this PR. Unsure if it's needed at all, I'll try removing it

Reply:
Looks like we need it